pkg/vpcmodel/nodesExplainability.go

```diff
@@ -10,6 +10,8 @@ import (
 
 var filterLayers = [2]string{NaclLayer, SecurityGroupLayer}
 
+const resourceTypeNetworkInterface = "NetworkInterface"
```

Comment:
can we avoid duplicating the const string for network interface kind? is it a more ibm-specific term to be used only by ibmvpc pkg rather than in the abstract vpcmodel pkg?

Reply:
also, the name of the new function `GetNetworkInterfaces`  hints there is an entity "network interfaces" not defined in this level.. this pkg is on level of abstract nodes